index.html

```diff
@@ -66,99 +76,103 @@ <h2>My Recent Works</h2>
       </div>
       <div class="projects">
         <img src="images/mainPH.svg" alt="img placeholder" />
-        <h2>Multi-Post Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="tools">
-          <li><button class="toolsBtn">CSS</button></li>
-          <li><button class="toolsBtn">HTML</button></li>
-          <li><button class="toolsBtn">Bootstrap</button></li>
-          <li><button class="toolsBtn">Ruby</button></li>
-        </ul>
-        <button class="mainSeeBtn">See Project</button>
-      </div>
-      <!-- Card 1-->
-      <div class="cardContainer">
-        <div class="card">
-          <h2>Profesional Art <br />Printing Data</h2>
-          <p>
+        <div class="">
+          <h2 class="projectTitle">Multi-Post Stories</h2>
+          <p class="projectDesc">
             A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
           </p>
-          <ul class="cardBtns">
-            <li><button class="cardBtn">HTML</button></li>
-            <li><button class="cardBtn">BootStrap</button></li>
-            <li><button class="cardBtn">Ruby</button></li>
+          <ul class="tools">
+            <li><button class="toolsBtn">CSS</button></li>
+            <li><button class="toolsBtn">HTML</button></li>
+            <li><button class="toolsBtn">Bootstrap</button></li>
+            <li><button class="toolsBtn">Ruby</button></li>
           </ul>
+          <button class="mainSeeBtn">See Project</button>
         </div>
-        <button class="cardSeeProj">See Project</button>
       </div>
-      <!-- Card 1-->
-      <div class="cardContainer">
-        <div class="card">
-          <h2>Profesional Art <br />Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard
-          </p>
-          <ul class="cardBtns">
-            <li><button class="cardBtn">HTML</button></li>
-            <li><button class="cardBtn">BootStrap</button></li>
-            <li><button class="cardBtn">Ruby</button></li>
-          </ul>
+      <div class="cards">
```

Comment:
**1-** Currently, you have **5 cards** but the design has **6 cards** please consider adding one to the project.

**2-** Moreover, in the template, there exist 3 cards in a single row. But in your case in depends upon the width of the screen so on smaller screens it works well but when we go to bigger screens there can be 4 and 5 cards on a  single row. Please consider minimizing the number of cards per row to be 3 in accordance with the design.

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220549970-43af01ac-be5b-49be-bee7-343f47d0e81e.png)


## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220550110-b6fabad8-0a62-4b53-b7e1-2e819f642ef0.png)